### PR TITLE
Fix `force` flag on upgrade

### PR DESCRIPTION
**Pull request by stefee**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

The `force: true` flag is broken on `upgrade-helm-chart`.

### Description

Fix the broken `force: true` flag on `upgrade-helm-chart`.

### CODE REVIEW

File: src/commands/upgrade-helm-chart.yml [hunk 1]
```diff
@@ -85,7 +85,7 @@ parameters:
     default: false
   force:
     description: |
-      Use development versions, too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored.
+      Force resource updates through a replacement strategy.
```
Comment (stefee) [inline src/commands/upgrade-helm-chart.yml:88]: https://helm.sh/docs/helm/helm_upgrade/

**Comment by stefee** [comment]
Can I request review from codeowners please 👼🏻 @lokst @joosangkim @gmemstr

**Comment by joosangkim** [comment]
Hi @stefee I'm just one of contributors but why do you think `force` upgrade ?

**Comment by stefee** [comment]
The `force` option actually already exists on the orb, but currently doesn't do anything - which is probably just an oversight by original author. This PR implements the functionality for the `force` option as per Helm's documentation.

**Review by joosangkim — APPROVED** [review]

**Comment by gavinclarkeuk** [comment]
@KyleTryon adding you in as I see you've done recent merges on other orbs, and it looks like the owners of this orb are no longer at CircleCI. How do we go about getting this merged & released?

**Comment by brivu** [comment]
This has been added and merged in PR #70

**Pull request closed without merging**